config/helm/appmesh-controller/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: appmesh-controller
 description: App Mesh controller Helm chart for Kubernetes
-version: 1.12.3
-appVersion: 1.12.3
+version: 1.12.4
+appVersion: 1.12.4
```

Comment:
Hey Brandon, we typically self-manage version incrementation. Given this is changing a version, we'd likely cut a new minor version for it.